### DISCUSSION TITLE
🌏Please tell me nice place for a trip.

### DISCUSSION THREAD

**Discussion by gojiteji** [NAIST-SE/AttractingContributors#406] [General]
I'm thinking of going to somewhere for sightseeing.

I want to get some recommendation of nice places.

Please fork my repository( https://github.com/gojiteji/AttractingContributors-Trip/blob/main/README.md ) and give me pull requests!!

![](https://media0.giphy.com/media/lXC2gmHf2ypUs/giphy.gif)

**Comment by AtropsCooper** [comment]
My favorite added.
Help me please if you'd like to 🥺
https://github.com/AtropsCooper/AttractingContributors-Random

**Comment by aru1702** [comment]
I love traveling.
I hope to travel together someday.
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<
